src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/LanguageServerFeatureOptions.cs

```diff
@@ -39,4 +39,6 @@ internal abstract class LanguageServerFeatureOptions
     /// When enabled, design time code will not be generated. All tooling will be using runtime code generation.
     /// </summary>
     public abstract bool ForceRuntimeCodeGeneration { get; }
+
+    public abstract bool UseRoslynTokenizer { get; }
```

Comment:
nit: doc-comment?

Reply:
I think the name is self-explanatory enough. Most of the flags here don't have XML comments, and in fact the one that is shown in the diff here in GitHub is incorrect anyway :)